Taarafo.Core/Brokers/Storages/StorageBroker.cs

```diff
@@ -30,6 +31,8 @@ private async ValueTask<T> InsertAsync<T>(T @object)
 			return @object;
 		}
 
+		private IQueryable<T> SelectAll<T>() where T : class => this.Set<T>();
```

Comment:
add `optionsBuilder.UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking);` to `protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)`